pulsar-broker/src/main/java/org/apache/pulsar/broker/service/SystemTopicBasedTopicPoliciesService.java

```diff
@@ -234,6 +233,7 @@ private void prepareInitPoliciesCache(NamespaceName namespace, CompletableFuture
             CompletableFuture<SystemTopicClient.Reader<PulsarEvent>> readerCompletableFuture =
                     creatSystemTopicClientWithRetry(namespace);
             readerCaches.put(namespace, readerCompletableFuture);
+            ownedBundlesCountPerNamespace.putIfAbsent(namespace, new AtomicInteger(1));
```

Comment:
If we move this line here, can we ensure the `ownedBundlesCount` will decrease to 0 after all the bundles of the namespace are unloaded from this broker?

Reply:
yes. This is only a init operation, we need to keep `readerCaches` and `ownedBundlesCountPerNamespace`  the data consistent.

Reply:
Oh, I see. My mistake.